convert_models/README.md

```diff
@@ -33,12 +33,12 @@ streamlit run convert_models/ui.py
 
 ## Model Storage and Use
 
-This process writes the models into a Podman volume under a `gguf/` directory and not directly back to the user's host machine (This could be changed in an upcoming update if it is required).
+This process writes the models into a podman volume under a `gguf/` directory and not directly back to the user's host machine (This could be changed in an upcoming update if it is required).
 
-If a user wants to access these models to use with the llamacpp model-service, they would simply point their model-service volume mount to the Podman volume created here. For example:
+If a user wants to access these models to use with the llamacpp_python model server, they would simply point their model service to the correct podman volume at run time. For example:
 
 ```
-podman run -it -p 8001:8001 -v models:/locallm/models:Z -e MODEL_PATH=models/gguf/<MODEL_NAME> -e HOST=0.0.0.0 -e PORT=8001 llamacppserver
+podman run -it -p 8001:8001 -v models:/locallm/models:Z -e MODEL_PATH=models/gguf/<MODEL_NAME> -e HOST=0.0.0.0 -e PORT=8001 llamacpp_python
```

Comment:
the standard model path has also changed

Reply:
good catch, fixing

Reply:
`/opt/app-root/src/converter/converted_models/gguf/` is this the new default path?

Reply:
for llamacpp its `/locallm/models/granite-7b-lab-Q4_K_M.gguf` Makefiles are the source of truth on these things